src/statsmodel.jl

```diff
@@ -193,6 +193,22 @@ function StatsBase.coeftable(model::TableModels; kwargs...)
     ct
 end
 
+_show_fit_stats(io::IO, model::TableModels) = nothing
+
+function _show_fit_stats(io::IO, model::TableRegressionModel)
+    try
+        println("R²: ", round(r2(model), sigdigits=4),
+             "\t Adjusted R²: ", round(adjr2(model), sigdigits=4))
```

Comment:
This should ensure that 4 digits are always used, and that "Ajusted" is aligned with the value of the F-statistic on the next line.
```suggestion
        @printf(io, "R²: %.4f  Adjusted R²: %.4f", r2(model), adjr2(model))
```